examples/tsc/BUILD.gn

```diff
@@ -0,0 +1,201 @@
+# Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import("//build/toolchain/cc_wrapper.gni")
+import("//build_extra/flatbuffers/flatbuffer.gni")
+import("//build_extra/flatbuffers/rust/rust_flatbuffer.gni")
+import("//build_extra/rust/rust.gni")
+import("//third_party/v8/gni/v8.gni")
+import("//third_party/v8/snapshot_toolchain.gni")
+import("//libdeno/deno.gni")
+
+group("default") {
+  testonly = true
+  deps = [
+    ":tsc",
+  ]
+}
+
+main_extern = [
+  "$rust_build:ansi_term",
+  "$rust_build:atty",
+  "$rust_build:dirs",
+  "$rust_build:flatbuffers",
+  "$rust_build:futures",
+  "$rust_build:getopts",
+  "$rust_build:http",
+  "$rust_build:hyper",
+  "$rust_build:hyper_rustls",
+  "$rust_build:lazy_static",
+  "$rust_build:libc",
+  "$rust_build:log",
+  "$rust_build:rand",
+  "$rust_build:regex",
+  "$rust_build:remove_dir_all",
+  "$rust_build:ring",
+  "$rust_build:rustyline",
+  "$rust_build:serde_json",
+  "$rust_build:source_map_mappings",
+  "$rust_build:tempfile",
+  "$rust_build:tokio",
+  "$rust_build:tokio_executor",
+  "$rust_build:tokio_fs",
+  "$rust_build:tokio_io",
+  "$rust_build:tokio_process",
+  "$rust_build:tokio_threadpool",
+  "$rust_build:url",
+]
```

Comment:
Can you use "//:deno_deps" instead?

Reply:
I tried, but all I got was unfound crate issues when I tried to update the `extern` attribute on the rust build.